src/layers/core/path-layer/path-layer-vertex-64.glsl.js

```diff
@@ -133,8 +135,8 @@ vec3 lineJoin(vec2 prevPoint64[2], vec2 currPoint64[2], vec2 nextPoint64[2]) {
   offsetScale = mix(offsetScale, 1.0 / max(cosHalfA, 0.001), step(0.5, cornerPosition));
 
   // special treatment for start cap and end cap
-  float isStartCap = step(0.0, -lenA);
-  float isEndCap = step(0.0, -lenB);
+  float isStartCap = lenA < 1.0e-6 ? 1.0 : 0.0;
```

Comment:
Using 0.0 with the step function makes the startCap / endCap decision fluctuates in 64-bit mode. A small delta is chosen to make this decision more numerically stable. Is 1e-6 appropriate here? @Pessimistress